### PR TITLE
PCC-842, PCC-814, PCC-824 Added: Drupal import command.

### PR DESCRIPTION
**Pull request by kevinstubbs**

### CODE REVIEW

File: packages/cli/src/lib/addonApiHelper.ts [hunk 1]
```diff
@@ -49,14 +49,81 @@ class AddOnApiHelper {
...
+  static async updateDocument(
+    documentId: string,
+    siteId: string,
+    title: string,
+    tags: string[],
+    metadataFields: any,
+  ): Promise<Article> {
+    const idToken = await this.getIdToken();
+
+    console.log("update document", {
```
Comment (Aumkar) [inline packages/cli/src/lib/addonApiHelper.ts:105]: Please remove it

File: packages/cli/src/lib/addonApiHelper.ts [hunk 2]
```diff
@@ -49,14 +49,81 @@ class AddOnApiHelper {
...
+        contentType,
+        field: {
+          title: fieldTitle,
+          type: fieldType,
+        },
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${idToken}`,
+          "Content-Type": "application/json",
```
Comment (Aumkar) [inline packages/cli/src/lib/addonApiHelper.ts:90]: May be we can skip it since its the default

File: packages/cli/src/cli/commands/import.ts [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+            parents: [folderRes.data.id!],
+          },
+          media: {
+            mimeType: "text/html",
+            body: post.attributes.body.processed,
+          },
+        })) as GaxiosResponse<drive_v3.Schema$File>;
+
+        // Add it to the PCC site.
+        console.log("Get document", res.data.id);
```
Comment (Aumkar) [inline packages/cli/src/cli/commands/import.ts:136]: Please remove it if added for debugging

File: starters/nextjs-starter-ts/components/smart-components/index.ts [hunk 4]
```diff
@@ -10,6 +10,7 @@ export const serverSmartComponentMap = {
         displayName: "Title",
         required: true,
         type: "string",
+        defaultValue: "Some title",
```
Comment (Aumkar) [inline starters/nextjs-starter-ts/components/smart-components/index.ts:13]: Why do we need default value for Title field?

File: packages/cli/src/lib/addonApiHelper.ts (hunk 1 above)
Comment (kevinstubbs) [reply packages/cli/src/lib/addonApiHelper.ts:105]: Gated behind verbose flag now.

File: starters/nextjs-starter-ts/components/smart-components/index.ts (hunk 4 above)
Comment (kevinstubbs) [reply starters/nextjs-starter-ts/components/smart-components/index.ts:13]: It was accidentally included in the changes. Good catch.

**Pull request merged**